conf/gulp-tasks/library.gulpfile.js

```diff
@@ -28,18 +29,43 @@ exports.dev = function devJSBundle () {
 };
 
 /**
- * Creates a build task per locale, and combines them into a series task.
+ * Creates a build task for each provided language and combines them into a series task.
+ * This function also supports locales, but it is named to reflect the current use case
+ * of creating bundles for just languages.
+ * @param {Array<string>} languages
  * @returns {Promise<Function>}
  */
-exports.default = function defaultJSBundle () {
-  const localizedTaskPromises = BUILD_LOCALES.map(locale => {
-    const minifyTaskFactory = new MinifyTaskFactory(locale);
-    return createBundleTaskFactory(locale)
+function createJSBundlesForLanguages (languages) {
+  const localizedTaskPromises = languages.map(language => {
+    const minifyTaskFactory = new MinifyTaskFactory(language);
+    return createBundleTaskFactory(language)
       .then(bundleTaskFactory => createBundles(bundleTaskFactory, minifyTaskFactory));
   });
   return Promise.all(localizedTaskPromises).then(localizedTasks => {
     return new Promise(resolve => series(compileCSS, ...localizedTasks)(resolve));
   });
+}
+
+exports.default = function defaultJSBundles () {
```

Comment:
Would it make sense to call this `defaultLanguageJsBundles`? I'm trying to think of how we can more easily differentiate this from `devJsBundle`

Reply:
Ya, that works